packages/imtr/src/parser.ts

```diff
@@ -231,7 +231,10 @@ const getQuestions = (
     }
 
     result.id = rule.attributes.id; // TODO: generate our own hash for id's
-    if (rule[INTER_PRIORITY]) result.prio = rule[INTER_PRIORITY];
+    // Just pass prio from rule-definition here, we'll fix the prio when merging the permits together
+    result.prio = rule[INTER_PRIORITY];
```

Comment:
I believe in our implementation (in FL) the prio field is guaranteed.

Maybe also add a test for this.

Reply:
double-checked.